packages/sandpack-core/src/npm/dynamic/fetch-protocols/utils/tar-store.ts

```diff
@@ -26,7 +26,13 @@ export class TarStore implements FetchProtocol {
     tarContents.forEach(tar => {
       normalized[tar.name.replace(/^[^/]+/, '')] = {
         // TODO(@CompuIves): store buffers rather than strings for binary files
-        content: Buffer.from(tar.buffer).toString(),
+        content: Buffer.from(tar.buffer).toString(
```

Comment:
Maybe add a second todo comment, to both explain why we do this and that we want to default to binary format in the future.